CommandLineTool.yml

```diff
@@ -238,6 +239,31 @@ $graph:
         return a string or an array of strings, which will then be evaluated as
         one or more glob patterns.  Must only match and return files which
         actually exist.
+
+        If the value of glob is a relative path pattern (does not
+        begin with a slash '/') then it is resolved relative to the
+        output directory.  If the value of the glob is an absolute
```

Comment:
I suggest that we either let people glob from arbitrary locations, or only have glob expressions that are relative to the output directory. To simplify things, perhaps we should only allow relative globs (i.e. only products under the output directory are capturable)